src/bun.ts

```diff
@@ -1,20 +1,21 @@
 import { BunSqliteDialect } from "kysely-bun-sqlite";
+import { serveStatic } from "hono/bun";
 // @ts-ignore
 import * as bunSqlite from "bun:sqlite";
 import app from "../src/app";
 import { oAuth2ClientFactory } from "../src/services/oauth2-client";
 import createAdapters from "./adapters/kysely";
 import createEmailAdapter from "./adapters/email";
 import { getDb } from "./services/db";
-import { migrateToLatest } from "../migrate/migrate";
+
+app.use("/static/*", serveStatic({ root: "./" }));
 
 const server = {
   async fetch(request: Request): Promise<Response> {
     const dialect = new BunSqliteDialect({
       database: new bunSqlite.Database("db.sqlite"),
     });
     const db = getDb(dialect);
-    migrateToLatest(dialect);
```

Comment:
Running each time slows it down..

Reply:
I don't see any difference in the runtime on github. I'd expect a screenshot or before/after comparison. What developers think slows things down rarely if ever does.